docs/tools/sign-in-with-ethereum/README.md

```diff
@@ -10,8 +10,15 @@ In many contexts, your application does not need a full "web3" provider, but jus
 
 > "_Ethereum_" here does not refer to any network in particular but to the type of wallet that can used. Since Polygon, Gnosis Chain or Optimism for example are all using the same wallet, you can (should!) absolutely use the same "Sign In with Ethereum".
 
+> Yeah, the sign in with Ethereum seems confusing to make you think it's all about Ethereum only. No, it covers Web3 sign in as a whole not just Ethereum (though initially, it all started with Ethereum - no wonder the confusing naming).
+
```

Comment:
I love this change in language to be more friendly. As a rule we always say web3 with a little "w" for consistency.

Reply:
I have changed the Web3 to web3 as requested